spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -365,21 +365,6 @@
       expect(response).to render_template(:new)
     end
 
-    it 'logs Encryption::EncryptionError' do
```

Comment:
This was somehow passing even though `User` doesn't have an `unlock_user_access_key` method (I think since f4adcf34dd1fcc9164c54a5f4c6a6a6369bb4b4c)

Reply:
it's that `allow(user).to receive(:unlock_user_access_key)` LOL....stubbing 😭

Reply:
but it wouldn't be called to trigger the error that's expected either 😬